python/ray/tune/trainable.py

```diff
@@ -462,20 +459,22 @@ def _train(self):
 
         raise NotImplementedError
 
-    def _save(self, checkpoint_dir):
+    def _save(self, tmp_checkpoint_dir):
         """Subclasses should override this to implement save().
 
+        Do not rely on absolute paths in the implementation of `_save` and
+        `_restore`. Use `validate_save_restore(use_object_store=True)`
+        to catch _save/_restore errors before execution.
```

Comment:
cc @ericl, an attempt to avoid user errors...

Reply:
Why not auto-detect these issues? (check path.startswith("/"))

Also I have no idea what validate_save_restore does, probably should remove that.

Reply:
Users do `pickle.save({some_info: "absolute_path"})` as an implementation detail that does not go through Tune. 

Users have already stated benefits of `validate_save_restore`, so I'm not going to remove that. https://groups.google.com/d/msg/ray-dev/pOl26QImkv4/9GhKIzOQBQAJ

Reply:
Shouldn't this say how to verify it then, with an example?

Reply:
Oh I see, I thought you were pointing at the rst where there was an example; ok I’ll add that - thanks.